HelpSource/Classes/UnitTest.schelp

```diff
@@ -396,28 +396,30 @@ TestYourClass : UnitTest {
 
 	test_yourMethod {
 		// every method whose name begins with "test_" will be run
+		var synth;
 
 		this.assert( 6 == 6, "6 should equal 6");
 
 		this.assertEquals( 9, 9, "9 should equal 9");
 
-		this.assertFloatEquals( 4.0 , 1.0 * 4.0 / 4.0  4.0,
+		this.assertFloatEquals( 4.0, 1.0 * 4.0 / 4.0 * 4.0,
 			"floating point math should be close to equal");
 
 		// we are inside a Routine, you may wait
 		1.0.wait;
 
 		// this will wait until the server is booted
-		this.bootServer;
-
 		// if the server is already booted it will free all nodes
 		// and create new allocators, giving you a clean slate
-		p = { SinOsc.ar };
-		p.play;
-		p.register;
+		this.bootServer;
+
+		synth = { SinOsc.ar / 10 }.play;
+		synth.register; 
+
 		// will wait until the condition is true
 		// will be considered a failure after 10 seconds
-		this.wait( { p.isPlaying }, "waiting for synth to play", 10);
+		// (fails since register was called after synth was already playing)
```

Comment:
This comment doesn't seem to be true in this case. Since the synth takes time to start, it will be registered by that time. To check:
```
(
{
	~synth = { SinOsc.ar / 10 }.play;
	~synth.register; 
	s.sync;
	~synth.isPlaying.postln; // true
}.fork
)
```

Reply:
Done - thx!

Reply:
@dyfer bump!

Reply:
sorry/thanks for the bump!